build/Dockerfile

```
@@ -1,3 +1,3 @@
-FROM node:10
+FROM node:10.12
```

Could we use 10.22? I tried running [this](https://app.circleci.com/pipelines/github/mattermost/mattermost-webapp/24060/workflows/fbedbe88-c9cb-49fe-825d-7a96e7bf8379/jobs/154776) on 10.12 but was getting weird errors.

```suggestion
FROM node:10.22
```

---

Maybe it is a `npm cache clean --force` problem. You can also accept the PR to get the changes into your feature branch.

---

Sorry about that (I didn't realize at first this was targeting that branch.) Thanks for taking care of this!